query-node/mappings/src/content/nft.ts

```diff
@@ -550,6 +551,20 @@ export async function contentNft_AuctionBidMade({ event, store }: EventContext &
   // create record for winning bid
   const { member, video } = await createBid(event, store, memberId.toNumber(), videoId.toNumber(), bidAmount.toString())
 
+  // Ensure if planned auction period would be extended
+  if (extendsAuction.valueOf() === true) {
+    const { auction } = await getCurrentAuctionFromVideo(
+      store,
+      videoId.toString(),
+      'Non-existing video got new bid',
+      'Non-existing auction got new bid'
+    )
+
+    const englishAuctionExtensionPeriod = (auction.auctionType as AuctionTypeEnglish).extensionPeriod!
+    auction.plannedEndAtBlock = auction.plannedEndAtBlock! + englishAuctionExtensionPeriod
```

Comment:
While your solution here is valid, I'd suggest having default values `0` rather than `non-null assertion`. It is safer in case we introduce an inconsistency (a bug) in the future that will break the assertation. Saving `NaN`(`isNaN(undefined + 1) === true`) to database could cause a processor to crash; it would be better to have "just" an invalid value in such case.
```suggestion
    const englishAuctionExtensionPeriod = (auction.auctionType as AuctionTypeEnglish).extensionPeriod
    auction.plannedEndAtBlock = (auction.plannedEndAtBlock || 0) + (englishAuctionExtensionPeriod || 0)
```